engine/api/src/main/java/pl/touk/nussknacker/engine/api/validation/Literal.java

```diff
@@ -0,0 +1,12 @@
+package pl.touk.nussknacker.engine.api.validation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target(ElementType.FIELD)
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Literal {
```

Comment:
Maybe it would be overkill, but what about https://docs.oracle.com/javaee/7/api/javax/validation/constraints/Pattern.html ? it gives much more flexibility.

Reply:
Pattern is good idea, but I think it should be done as separeted Validator. It is easier use: `        @Literal(`type` = LiteralType.Integer)`.